main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapActivity.java

```diff
@@ -440,7 +440,12 @@ private void reloadCachesAndWaypoints() {
                 }, () -> {
                     if (viewport3.get() != null) {
                         if (setDefaultCenterAndZoom) {
-                            mapFragment.zoomToBounds(viewport3.get());
+                            final Geopoint storedMapCenter = Settings.getUMMapCenter();
```

Comment:
Shouldn't you apply this logic only if the new Setting `pref_autozoom_consider_lastcenter` is set to true?

Reply:
ooops, what a stupid mistake ... Adding a setting and then not using it :-(